clojure-mode.el

```diff
@@ -861,8 +861,8 @@ any number of matches of `clojure--sym-forbidden-rest-chars'."))
 
       ;; namespace definitions: (ns foo.bar)
       (,(concat "(\\<ns\\>[ \r\n\t]*"
-                ;; Possibly metadata
-                "\\(?:\\^?{[^}]+}[ \r\n\t]*\\)*"
+                ;; Possibly metadata, shorthand and/or longhand
+                "\\(?:\\^?\\(?:{[^}]+}\\|:[^ \r\n\t]+[ \r\n\t]\\)[ \r\n\t]*\\)*"
```

Comment:
Could we use `\s-` or `[:space:]` instead of `[ \r\n\t]`? Does it matter? It seems we use all three in different places throughout the code.

Reply:
Certainly, I just saw `[ \r\n\t]` being used and chose that. I'll change it and see if it causes any problems